ComSemApp/templates/ComSemApp/student/assessment.html

```diff
@@ -248,7 +248,7 @@ <h2 id="timer-text" class="show-expression"></h2>
   $("div." + classgroup).height("auto");
 
   var tallestcolumn = 0;
-  $("div." + classgroup).each(() => {
+  $("div." + classgroup).each(function() {
```

Comment:
This is weird that this created the spacing issue. I'm going to look into this

Reply:
I looked into it. The problem is on line 252 with the `this` in `$(this).height();`
Because anonymous functions are "compiled" before execution rather than being interpreted as they occur, this causes the change.
The JQuery documentation found [here](https://api.jquery.com/jQuery.each/) shows how the .each call should work. I was able to fix this issue by changing lines 251 and 252 to the following:
```
$("div." + classgroup).each((_, element) => {
    currentHeight = $(element).height();
```
I personally think we should go with this solution to keep arrow functions consistent over anonymous functions